sql/parse/parse.go

```diff
@@ -528,6 +528,30 @@ func convertShow(ctx *sql.Context, s *sqlparser.Show, query string) (sql.Node, e
 			node = plan.NewFilter(filter, node)
 		}
 		return node, nil
+	case "function status":
+		var filter sql.Expression
+
+		if s.Filter != nil {
```

Comment:
I'm not sure if this code is duplicated elsewhere. If so go ahead and factor out a function with this logic.